content/ember/v3/component-is-visible.md

```diff
@@ -1,21 +1,48 @@
----
-id: ember-component.is-visible
-title: Deprecate `Component#isVisible`
 until: '4.0.0'
 since: '3.15'
 ---
+In ember earlier versions, we had the concept of 'A controller should never call method or change property on its associated view '( Ember.View class)', instead the view should bind the state of its associated controller.'
+
+So, we were using something like this,
+```js
+App.MyView = Ember.View.extend({
+  isVisible: Ember.computed.alias('controller.myViewVisible')
+}
+```
+
+or in the component
+
+```js
```

Comment:
Would it make sense to add the context of the component into this example as well? f.e. by adding the import and export statements for the component class - what do you think?

Reply:
can you check now?

Reply:
This sounds great!